UlKit/UlKit/CartViewController.swift

```diff
@@ -3,9 +3,15 @@
 
 import UIKit
 
-final class CartViewController: UIViewController {
+class CartViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
-        title = "Корзина"
+    }
+}
+
+extension CartViewController: DetailsShoesViewControllerDelegate {
+    func sendInfo(_ from: DetailsShoesViewController, product: Product) {
```

Comment:
тот кто реализует протокол, в данном случае CartViewController ничего не отправляет, наоборот принимает
так что тут имя наверное лучше заменить на 
```suggestion
    func acceptInfo(_ from: DetailsShoesViewController, product: Product) {
```